src/main/docker/Dockerfile.jvm

```diff
@@ -77,7 +77,7 @@
 #   accessed directly. (example: "foo.example.com,bar.example.com")
 #
 ###
-FROM registry.access.redhat.com/ubi8/openjdk-17-runtime:1.20
+FROM registry.access.redhat.com/ubi8/openjdk-21-runtime:1.20
```

Comment:
We have 4 Dockerfiles, should probably update others. Or at least `Dockerfile-profiling.jvm`?

Reply:
yes, I missed that one. Fixed